.github/workflows/automate-changelog-PR.yml

```diff
@@ -0,0 +1,34 @@
+name: "Create PR with updated CHANGES.md"
+
+on:
+  workflow_dispatch:
+    inputs:
+      fname_changelog:
+        description: 'Changelog filename'
+        required: true
+        default: 'CHANGES.md'
+
+jobs:
+  create-changelog-PR:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Install changelog
+      run: pip install changelog-neuropoly
+    - name: Generate changelog
+      run: |
+        echo -e "MILESTONE=$(changelog neuropoly/spinalcordtoolbox --update --name ${{ github.event.inputs.fname_changelog }} | grep 'INFO Milestone' | head -1 | cut -d' ' -f3 | cut -d',' -f1)" >> ${GITHUB_ENV}
```

Comment:
This is a bit of trickery to get the milestone name from the stdout of `changelog`. Perhaps changelog could be updated to make this easier? Or, perhaps there's an easier way already -- suggestions appreciated!

Reply:
IMO a better solution would be to explicitly specify the milestone name. There's an open feature request for this here: https://github.com/neuropoly/changelog/issues/15

Explicitly specifying the name would also avoid the issue described in https://github.com/neuropoly/changelog/issues/22.

Reply:
Done in https://github.com/neuropoly/changelog/pull/23.

Reply:
Merged upstream, applied to this PR in d9a90f3.